src/ray/stats/stats.h

```diff
@@ -104,7 +90,6 @@ static inline void Init(
   opencensus::stats::DeltaProducer::Get()->SetHarvestInterval(
       StatsConfig::instance().GetHarvestInterval());
 
-  MetricPointExporter::Register(exporter, metrics_report_batch_size);
```

Comment:
IIRC i remember u had a PR that changes this?  @rkooo567 
So this was not used?